EmotionCalculator/EmotionCalculator/FormsUI/BaseForm.cs

```diff
@@ -125,12 +130,22 @@ private void CameraStopButton_Click(object sender, EventArgs e)
 
         private async void SubmitWebCamButton_Click(object sender, EventArgs e)
         {
-            Image image = null;
+            if (!ConnectionChecker.IsConnectedToInternet())
```

Comment:
ok tai siūlau error išmest useriui "internet connection is required" ar sth

Reply:
Can do that